### ISSUE TITLE
MP GO:0031362   anchored component of external side of plasma membrane

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#14948]
our annotations to
GO:0031362   anchored component of external side of plasma membrane
seem to have lost their "cell surface" annotation over the weekend. This means the "cell surface" parent must have been removed?

was that intended?
I can't see exactly what was changes as this parent is still visible in Quickgo.

val

**Comment by pgaudet** [comment]
Hi Val, 

The parent is there (part_of) 

<img width="523" alt="image" src="https://user-images.githubusercontent.com/4782928/35525053-b40c95ce-0523-11e8-837d-5ce799bd1893.png">


Are we seeing the same ? (The relation seems wrong to the ontology developers, however) 

Thanks, Pascale

**Comment by ValWood** [comment]
It's wierd,

i have 10 genes annotated to "anchored component of external side of plasma membrane"
which are no longer annotated to "cell surface"

(they were annotated to cell surface last week, because I compared  our dev version to live on Sun. Now  both dev and live have the lower number)

![cell surface](https://user-images.githubusercontent.com/7359272/35528904-37f09644-0527-11e8-90fe-a643984ba790.jpg)

**Comment by ValWood** [comment]
>The relation seems wrong to the ontology developers, however
 
we are using the term "cell surface" in accordance with this comment

Comments
Note that this term is intended to annotate gene products that are attached (integrated or loosely bound) to the plasma membrane or cell wall.

I'm happy for any of this to change, but  everyone calls these "tail anchored" proteins (both GPI or membrane anchored) "cell surface proteins. So it seems like the correct term.

Basically we  need  a term to represent everything which is *outside* the cell (cell wall or plasma membrane attached). It's a largely biologically cohesive module (with some sub divisions) .... I'm happy to use any other term which fits this though....

**Comment by ukemi** [comment]
What about cells with cell walls?

**Comment by ValWood** [comment]
Fungi have cell walls- but in fungi "cell surface" is used for anything outside of the plasma membrane...AFAIK

cell surface =orange below 

![cell surface at pombase](https://user-images.githubusercontent.com/7359272/35532156-34ed3cf4-0531-11e8-91b9-9c9f9154773a.jpeg)

**Comment by ValWood** [comment]
for us anything outside of the cell is on the surface....

**Comment by pgaudet** [comment]
Hi @ValWood 
Is the parent still missing ?

**Comment by ValWood** [comment]
no it  they came back;)
